aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,29 +23,67 @@ import { Example } from '../../layouts';
   <SimpleSignUpExample />
 </Example>
 
-## Guidance
+### Jump to section...
+[Use cases and example code](#when-to-use)  
+[Do's and Don'ts visual examples](#Dos)  
```

Comment:
since we dont have visuals in this page should we change this name?

Reply:
We are working on this. I'm trying to get the text approved first, then we can work on the visuals.